### PR TITLE
feat: alert and button component added

### PR DESCRIPTION
**Pull request by devdeadviz**
## Screenshots:
Screenshot (123)

Screenshot (122)

## Preview Deploy Link: https://deploy-preview-2--arcui.netlify.app/

### CODE REVIEW

File: pages/components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  margin: 1rem 0;
+  border-radius: 10px;
+  width: max-content;
+}
+
+/* Standard Alert */
+
+.alert-primary {
+  background-color: var(--primary-color);
+  color: #fff;
```
Comment (gkanishk) [inline pages/components/alert/alert.css]: try to add `#fff` inside variables to make everything generic

**Review by gkanishk — CHANGES_REQUESTED** [review]
Everything looks to me, please check responsiveness and resolve the overflow issue
Screenshot 2022-02-02 at 8 18 13 PM

File: pages/components/alert/alert.css (hunk 1 above)
Comment (devdeadviz) [reply pages/components/alert/alert.css]: Added the changes.

**Review by bishnoimukesh — APPROVED** [review]
seems good to merge

**Review by xii-vi — APPROVED** [review]
You can merge.

**Pull request merged**